tests/step-023.sh

```diff
@@ -21,22 +21,22 @@ mkdir -p /tmp/shared/hooks-023-b.git/pre-commit &&
     exit 1
 
 # change it and expect it to change it back
-git config --global githooks.shared /tmp/shared/some-previous-example
+git config --global githooks.shared file:///tmp/shared/some-previous-example
```

Comment:
Anything `git config ... githooks.shared` shouldn't need to change to include the `file://`, right?
Should we change them all back?
I'd rather have one (or more) new tests testing for the new behaviour with local paths and local file URLs.

Reply:
^ this applies to all tests, won't comment on each one-by-one.

Reply:
Ok, I change them back. The ones which I can change back